src/main/resources/ttc.properties

```diff
@@ -94,6 +93,9 @@ de.konfidas.ttc.messages.systemlogs.logoutCauseNotFound = Fehler beim Parsen des
 de.konfidas.ttc.messages.systemlogs.roleNotFound = Fehler beim Parsen des systemOperationDataContent. Das Pflichtfeld role wurde nicht gefunden.
 de.konfidas.ttc.messages.systemlogs.timeAfterUpdateNotFound = Fehler beim Parsen des systemOperationDataContent. Das Pflichtfeld timeAfterUpdate wurde nicht gefunden
 de.konfidas.ttc.messages.systemlogs.timeBeforeUpdateNotFoud = Fehler beim Parsen des systemOperationDataContent. Das Pflichteil timeBeforeUpdate wurde nicht gefunden
+de.konfidas.ttc.messages.systemlogs.errorEventNameNotFound = Fehler beim Parsen des systemOperationDataContent. Das Pflichteil eventName wurde nicht gefunden
+de.konfidas.ttc.messages.systemlogs.errorEnabledNotFound = Fehler beim Parsen des systemOperationDataContent. Das Pflichteil enabled wurde nicht gefunden
+de.konfidas.ttc.messages.systemlogs.errorLoggingParameterUnknownTag = Fehler beim Parsen des systemOperationDataContent. Das Tag ist unbekannt
```

Comment:
errorLoggingParameterUnknownTag != systemOperationDataContent (something is wrong here)